spec/migrations/1_create_tables.rb

```diff
@@ -0,0 +1,35 @@
+Sequel.migration do
```

Comment:
We'll probably want to call this something more specific, but renames of the migration files don't affect the integer migrator.